tomls/governance/snapshot-record.toml

```
@@ -0,0 +1,17 @@
+[invoke.governance_set_snapshot_record_ambassador]
```

this is a very minor comment and by no means a blocker, but the very neat convention of naming invokes is `ContractName_function_MaybeSomeArgs`
In this case it would become 

```suggestion
[invoke.GovernanceProxy_setSnapshotContract_ambassador]
```
Because of pure copypaste it is easier to fill in (no thinking and manual case converions), it also makes it easier to read and understand in cannon state, it also sorts all invokes nicely grouped by contract/method